src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -188,6 +204,34 @@ private void saveWindowState(Stage mainStage) {
         Globals.prefs.putDouble(JabRefPreferences.POS_Y, mainStage.getY());
         Globals.prefs.putDouble(JabRefPreferences.SIZE_X, mainStage.getWidth());
         Globals.prefs.putDouble(JabRefPreferences.SIZE_Y, mainStage.getHeight());
+        printWindowState(mainStage);
+    }
+
+    /**
+     * outprints the Data from the Screen
+     * (only in debug mode)
+     * @param mainStage
+     */
+    private void printWindowState(Stage mainStage) {
+        StringBuilder bob = new StringBuilder();
+        bob.append("SCREEN DATA:");
+        bob.append("JabRefPreferences.WINDOW_MAXIMISED: " + mainStage.isMaximized() + "\n");
+        bob.append("JabRefPreferences.POS_X: " + mainStage.getX() + "\n");
+        bob.append("JabRefPreferences.POS_Y: " + mainStage.getY() + "\n");
+        bob.append("JabRefPreferences.SIZE_X: " + mainStage.getWidth() + "\n");
+        bob.append("JabRefPreferences.SIZE_Y: " + mainStage.getHeight() + "\n");
+        LOGGER.debug(bob.toString());
+    }
+
+    /**
+     * Tests if the window coordinates are out of the mainscreen
+     * @return outbounds
+     */
+    private boolean testExternalCoordinates() {
```

Comment:
`test` is a bit vague, what about `storedWindowPositionIsOutOfBounds` (or maybe something a bit shorter).

Reply:
renamed it with *isWindowPositionOutOfBounds()*